### PR TITLE
Color Highlighting for Reference documentation

### PR DESCRIPTION
**Pull request by Temidayo32**
This pull request compliments this pull request #230 and further attends to this issue #228.

### CODE REVIEW

File: sass/_component_autodoc.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+dd {
+    margin: 0 0 12px 24px;
+}
```
Comment (lb-) [inline sass/_component_autodoc.scss:3]: As discussed on the other PR we should ensure we can scope this whole CSS section to a root class that is only use by autodoc.

```
dl.class {
  .sig-object + dd {
    margin: 0 0 0.5rem 1.5rem;
  }
}

```

`.class` is the best I can find so lets also scope to the element.

**Review by lb- — CHANGES_REQUESTED** [review]
@Temidayo32  this is a great start but needs some solid work to get this in.

Here is a bit of a checklist
* [ ] Ensure you can run linting/formatting (sccss) locally, otherwise the CI will fail
* [ ] All colours should use the variables we already have in place, instead of hard-coding the values - see sass/_variables.scss for existing variables
* [ ] Avoid sizing too much by pixels, try to look at existing code to see what we use for padding, if needed use REMs
* [ ] I think we can find a nicer way to set up the classes instead of the extensive usage of `not`, hopefully if you scope all the css within the `dl.class {` you should find it works easier)
* [ ] When I ran this locally the dark mode 'source' link did not work correctly, it was not visible, be sure to check both theme usages.

Screen Shot 2022-10-27 at 6 40 41 am

File: sass/_component_autodoc.scss (hunk 1 above)
Comment (Temidayo32) [reply sass/_component_autodoc.scss:3]: Ok, I will make these changes and get back to you soonest

Comment (lb-) [reply sass/_component_autodoc.scss:3]: just a reminder that this specific change at the start of the file is already in the code now. So you will need to rebase on main or maybe merge main into this code.

**Comment by lb-** [comment]
Replaced with https://github.com/wagtail/sphinx_wagtail_theme/pull/235

**Pull request closed without merging**